packages/components/src/tools-panel/tools-panel-item/hook.js

```diff
@@ -56,6 +64,10 @@ export function useToolsPanelItem( props ) {
 	// Determine if the panel item's corresponding menu is being toggled and
 	// trigger appropriate callback if it is.
 	useEffect( () => {
+		if ( isResetting ) {
```

Comment:
Should `isResetting` be added to the dependencies of the `useEffect` hook?

Reply:
✅ Fixed. Thanks.